Source: @pullmoll
Upstream: no
Reason: The binutils macros for bfd section access changed slightly.

--- tools/perf/util/srcline.c	2020-05-20 08:20:41.000000000 +0200
+++ tools/perf/util/srcline.c	2020-05-27 13:48:11.606027082 +0200
@@ -197,7 +197,7 @@
 	if (a2l->found)
 		return;
 
-	if ((bfd_get_section_flags(abfd, section) & SEC_ALLOC) == 0)
+	if ((bfd_section_flags(section) & SEC_ALLOC) == 0)
 		return;
 
 	pc = a2l->addr;
@@ -201,8 +201,8 @@
 		return;
 
 	pc = a2l->addr;
-	vma = bfd_get_section_vma(abfd, section);
-	size = bfd_get_section_size(section);
+	vma = bfd_section_vma(section);
+	size = bfd_section_size(section);
 
 	if (pc < vma || pc >= vma + size)
 		return;
